src/components/form-components.js

```diff
@@ -110,10 +110,21 @@ export class InputField extends Component {
         this.setState({ error: 'This field should be a JSON object. JSON objects begin with a { and end with a }' });
       }
     } else if (format === 'xpub') {
```

Comment:
could we also change this to `pub` instead of `xpub`? an annoying refactor but I think it's a lot more maintainable since Stellar keys aren't BIP32 xpubs